Assets/Scripts/Reticule.cs

```diff
@@ -1,73 +1,63 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+﻿using UnityEngine;
 using UnityEngine.UI;
 
 public class Reticule : MonoBehaviour
 {
-    public SpriteRenderer crosshairsRenderer;
-    public Sprite crosshairsSprite;
-    public GameObject cannonPrefab;
+    public Transform Crosshairs;
     public Button button;
-    public GameObject l;
-    public GameObject lazer;
+    public Image EnergyFill;
+    public float FillSpeed;
```

Comment:
Either stick to the current class naming or entirely change it.

Reply:
DOne